app/demo-loop/LDemoIO.cc

```diff
@@ -209,6 +209,16 @@ TransporterInput load_input(const LDemoArgs& args)
                        << "' (expected gdml or root)");
     }
 
+    // Unconditionally add celerino pseudoparticle to imported data
+    ImportParticle particle;
+    particle.name      = "celerino";
+    particle.pdg       = 81;
+    particle.mass      = 0.0;
+    particle.charge    = 0.0;
+    particle.lifetime  = -1.0;
+    particle.is_stable = true;
+    imported_data.particles.push_back(particle);
```

Comment:
Something like:

```cpp
if (args.add_pseudoparticles) // or whatever name is preferred
{
    ... Create/Add Celertino(s)
}
```

or is there a better location?

Reply:
On second thoughts, maybe it's better to allow the `PhysicsParams` to accept an empty process list, and if so then to skip the appropriate kernels... it would definitely be simpler than adding new processes and such.

Reply:
OK, thanks for the update. I think we can definitely relax the restriction for cutoffs (allowing no cuts to be present), but based on the cascade of debug failures, I think adding a `NullProcess`/`Model` is indeed the way to go. Thanks for exploring that side trail and sorry it didn't work as I'd hoped!

Reply:
@drbenmorgan An update for you on this effort: in order to replicate Geant4 single-process results, @stognini is going to have to a `NullProcess/Model` which will have zero cross section (and null-op `execute` arguments) and which we'll construct with an input set of particle IDs. If you're interested in trying to tackle this next we can perhaps split up the work...

Reply:
@sethrj, that sounds good, and happy to help out with this. Keep me in the loop on plans/discussions, and can take on task(s) as needed to assist!